tests/ui/associated-types/associated-types-ICE-when-projecting-out-of-err.stderr

```diff
@@ -1,14 +1,8 @@
 error[E0277]: the trait bound `(): Add<A>` is not satisfied
-  --> $DIR/associated-types-ICE-when-projecting-out-of-err.rs:23:11
+  --> $DIR/associated-types-ICE-when-projecting-out-of-err.rs:15:11
    |
 LL |     r = r + a;
    |           ^ the trait `Add<A>` is not implemented for `()`
-   |
-help: this trait has no implementations, consider adding one
-  --> $DIR/associated-types-ICE-when-projecting-out-of-err.rs:15:1
-   |
-LL | trait Add<RHS=Self> {
-   | ^^^^^^^^^^^^^^^^^^^
```

Comment:
Problem [DIAGNOSTICS 2/2]: for instance, this stderr changed, probably because minicore's stubs don't have rustc diagnostics attributes (`#[diagnostics::*]`, `#[rustc_diagnostic_item = "..."]`, etc.).

Reply:
I don't think this stderr change is because of the lack of diagnostic attributes, just that there are other impls of `Add` in minicore that weren't in the manual stubs. This is an ICE regression test (it says in the name), so I think it's fine to change.

Reply:
Ahh right. Agreed.